sql/mob_groups.sql

```diff
@@ -8823,7 +8823,7 @@ INSERT INTO `mob_groups` VALUES (8771,233,167,7920,0,3445,0,0,56,62,0);
 INSERT INTO `mob_groups` VALUES (8772,461,167,1056,0,334,0,0,65,65,0);
 INSERT INTO `mob_groups` VALUES (8773,482,167,7920,128,0,7500,0,80,80,0);
 INSERT INTO `mob_groups` VALUES (8774,908,167,1056,0,636,0,0,52,54,0);
-INSERT INTO `mob_groups` VALUES (8775,1118,167,7920,128,798,0,0,72,73,0);
+INSERT INTO `mob_groups` VALUES (8775,1118,167,0,128,798,0,0,72,73,0);
```

Comment:
Looks like you hit `Drexerion the Condemned` instead of `Bodach`

Reply:
lol wtf